### DISCUSSION TITLE
Resize problem ?

### DISCUSSION THREAD

**Discussion by xZourSx** [sourcelair/xterm.js#4399] [Q&A]
Hello,

I have set up xterm.js however when I use it, my commands do not take the whole window but only a small part.

Do you know why?
Thanks
![Sans titre](https://user-images.githubusercontent.com/61470872/216657612-da2348a6-a61c-45d2-b28c-69892c8b9281.png)

**Comment by Tyriar** [comment]
This is typically caused by your pty not being sized correctly and/or it not matching the size of xterm.js
